hack/update-codegen.sh

```diff
@@ -18,25 +18,29 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_PACKAGE=k8s.io/cluster-registry
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Isn't this equivalent to just running `"$( dirname "${BASH_SOURCE[0]}" )"`? Is this to expand relative path to absolute path?

Reply:
Yes, that's the idea. Under the right circumstances, they should be equivalent, but I prefer to make sure the path is an absolute path.

Reply:
Ok. This is usually achieved through `readlink -f`, but that also follows symlinks. What you have now works too.